AdobeStockImageAdminUi/view/adminhtml/layout/cms_wysiwyg_images_index.xml

```diff
@@ -7,7 +7,7 @@
 -->
 <layout xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/layout_generic.xsd">
     <referenceContainer name="root">
-        <block class="Magento\Backend\Block\Template" template="Magento_AdobeStockImageAdminUi::panel.phtml" name="stock.panel" aclResource="Magento_AdobeStockImageAdminUi::save_preview_images">
+        <block class="Magento\AdobeStockImageAdminUi\Block\Adminhtml\Panel" template="Magento_AdobeStockImageAdminUi::panel.phtml" name="stock.panel" aclResource="Magento_AdobeStockImageAdminUi::save_preview_images">
```

Comment:
Please move signIn functionality to a child block of Panel to separate the logic

Reply:
ok